v-next/hardhat-keystore/src/internal/tasks/get.ts

```diff
@@ -0,0 +1,44 @@
+import type { KeystoreLoader } from "../types.js";
+import type { UserInteractions } from "../ui/user-interactions.js";
+import type { HardhatRuntimeEnvironment } from "@ignored/hardhat-vnext/types/hre";
+import type { NewTaskActionFunction } from "@ignored/hardhat-vnext/types/tasks";
+
+import { setupDirectInterruptionsAndKeystoreLoader } from "../utils/setup-direct-interruptions-and-keystore-loader.js";
+
+interface TaskGetArguments {
+  key: string;
+}
+
+const taskGet: NewTaskActionFunction<TaskGetArguments> = async (
+  { key },
+  hre: HardhatRuntimeEnvironment,
+): Promise<void> => {
+  const { keystoreLoader, interruptions } =
+    await setupDirectInterruptionsAndKeystoreLoader(hre);
+
+  await get({ key }, keystoreLoader, interruptions);
+};
+
+export const get = async (
+  { key }: TaskGetArguments,
+  keystoreLoader: KeystoreLoader,
+  interruptions: UserInteractions,
+): Promise<void> => {
+  if (!(await keystoreLoader.isKeystoreInitialized())) {
+    process.exitCode = 1;
+    return interruptions.displayNoKeystoreSetErrorMessage();
```

Comment:
This is not an interruption, it's normal output coming from the main and only task being executed at this point in time.

Same for the rest.

The only real interruption from this plugin should be "hey, i need the password to the keystore", as that doesn't come from a task, but from a hook.